mqtt/mqtt-bridge/src/persist/loader.rs

```diff
@@ -134,11 +132,12 @@ mod tests {
         };
 
         // insert elements
-        let mut state_lock = state.lock();
-        state_lock.insert(key1, pub1.clone()).unwrap();
-        state_lock.insert(key2, pub2).unwrap();
-        drop(state_lock);
-
+        let key1;
```

Comment:
I don't have objections about this syntax, but just curious how is this better than inline declaration + drop(lock)?

Reply:
It was more of an easy pattern to copy paste, as I do the same in for loops in other parts of the code since keys are returned from inserts now. I have no preference on this, I just copied over what I already started with, if want the other way I will change :)